core/shared/src/main/scala/zio/prelude/ZSet.scala

```diff
@@ -396,3 +402,32 @@ trait LowPriorityZSetImplicits {
     )
 
 }
+
+trait ZSetSyntax {
+  implicit final class ZSetMapOps[+A](self: Map[A, Natural]) {
+
+    /** Converts a `Map[A, Int]` to a `MultiSet` */
+    def toMultiSet[A1 >: A]: MultiSet[A1] = MultiSet.fromMap(self)
```

Comment:
Btw @adamgfraser are these `[A1 >: A]` necessary for for methods that don't take have any parameters and only return values?

Reply:
You don't need it if `A` just appears as an output like here.